Atlantic 2023 website (#1902)
writethedocs/www: docs/include/conf/events/writing-day.rst

> it pullEach year we run an event called the Writing Day during the conference. This event is modeled after the concept of `sprints <http://en.wikipedia.org/wiki/Sprint_%28software_development%29>`_, which are common in open-source conferences.

Reviewer: Errant `git pull`

```suggestion
Each year we run an event called the Writing Day during the conference. This event is modeled after the concept of `sprints <http://en.wikipedia.org/wiki/Sprint_%28software_development%29>`_, which are common in open-source conferences.
```
